### ISSUE TITLE
Can i add Simple website using only HTML and CSS

### ISSUE THREAD

**Issue by Kedarnath-Rothe** [you-dont-need/You-Dont-Need-JavaScript#359]

**Comment by aayush105** [comment]
@Kedarnath-Rothe yes you can work on it and provide the output in description and image folder.

**Comment by Sejal99** [comment]
Yes, you can create simple websites using HTML and CSS. In fact, HTML (Hypertext Markup Language) and CSS (Cascading Style Sheets) are the fundamental technologies for building web pages. Here's a brief overview of how you can create a basic website using these technologies:

**Comment by Tabindanoor** [comment]
yes it can be done obviously
please assign it to me

**Comment by aayush105** [comment]
> yes it can be done obviously please assign it to me

Create a PR if you want to contribute @Tabindanoor , we will review it and merge it. And please read CONTRIBUTING.md file before create a PR.

**Comment by srishtiagarwall** [comment]
Greetings, Sir @aayush105 I would like to collaborate with you on this issue.

**Comment by aayush105** [comment]
> Greetings, Sir @aayush105 I would like to collaborate with you on this issue.
Sure @srishtiagarwall we can work.

**Comment by aayush105** [comment]
> Greetings, Sir @aayush105 I would like to collaborate with you on this issue.

Sure @srishtiagarwall we can work.

**Comment by srishtiagarwall** [comment]
> Greetings, Sir @aayush105 I would like to collaborate with you on this issue.
>
> Sure @srishtiagarwall, we can work.
>
Thank you, @aayush105! Can you specify the criteria and requirements for the issue so that I can proceed further?

**Comment by aayush105** [comment]
> > Greetings, Sir @aayush105 I would like to collaborate with you on this issue.
> > Sure @srishtiagarwall, we can work.
> 
> Thank you, @aayush105! Can you specify the criteria and requirements for the issue so that I can proceed further?

@srishtiagarwall the requirement is simple just build a simple website using HTML and CSS only and don't use JS.

**Comment by Garv0504** [comment]
Greetings, Sir **@aayush105** I would like to know whether this issue is open for a PR or not because I have made a simple website using just HTML and CSS. I would love to collaborate with you and @srishtiagarwall on this issue.

**Comment by aayush105** [comment]
> Greetings, Sir **@aayush105** I would like to know whether this issue is open for a PR or not because I have made a simple website using just HTML and CSS. I would love to collaborate with you and @srishtiagarwall on this issue.

hello @Garv0504 , yes you can contribute to the repo by adding you website, i will go through is and merge it. Also read the contributing.md file for the proper knowledge how to contribute and where to create a folder of you project and also add the output image in the README.md as well as in the description while create a PR.

**Comment by Inija-2503** [comment]
Greetings,@aayush105 i would like to work on this issue as i know how to make a website using html and css

**Comment by aayush105** [comment]
> Greetings,@aayush105 i would like to work on this issue as i know how to make a website using html and css

@Inija-2503 yes you can work on it as well !! just follow the guideline.

**Comment by Omowunmikamil** [comment]
Good morning, @aayush105 is this issue still open for contribution?

**Comment by aayush105** [comment]
> Good morning, @aayush105 is this issue still open for contribution?

Yes, you can contribute by making websites. @Omowunmikamil

**Comment by Garv0504** [comment]
Greetings @aayush105 I have successfully added a website made of HTML CSS kindly check the PR #460. I am open for any kind of changes if applicable.

**Comment by Inija-2503** [comment]
> > Greetings,@aayush105 i would like to work on this issue as i know how to make a website using html and css
> 
> @Inija-2503 yes you can work on it as well !! just follow the guideline.

Greetings,@aayush105 i have added a website kindly check the PR for the issue #359 . i am open for any  sort of suggestion ,Thank you

**Comment by aayush105** [comment]
> > > Greetings,@aayush105 i would like to work on this issue as i know how to make a website using html and css
> > 
> > 
> > @Inija-2503 yes you can work on it as well !! just follow the guideline.
> 
> Greetings,@aayush105 i have added a website kindly check the PR for the issue #359 . i am open for any sort of suggestion ,Thank you

Sure @Inija-2503

**Comment by Garv0504** [comment]
Dear @aayush105 kindly check my new PR #487 where I have added a new website based on HTML and CSS. If there will be a requirement for any kind of change then I will be ready for that also.

**Comment by Kirtibisht11** [comment]
hey @aayush105 i'm interested in this issue. can i contribute?

**Comment by aayush105** [comment]
> hey @aayush105 i'm interested in this issue. can i contribute?

sure you can @Kirtibisht11

**Comment by Maralsabetian** [comment]
Hello, 
I have a project assigned by my prof. to contribute on github projects.
It is my first time and I am familiar with html and css.
Can I contribute or it is complete?
can you please guide me a little?

**Comment by Kepvion** [comment]
Hey,
@aayush105 
I’d like to work on this issue and create a PR.

**Comment by aayush105** [comment]
sure @Wanderer65 you can work on this.

**Comment by pranavpanmand** [comment]
Hi @aayush105,
I’d like to work on this issue. I can create a simple website using only HTML and CSS.  #359

**Comment by Kepvion** [comment]
Just a website??any

**Comment by pranavpanmand** [comment]
Greetings @aayush105 👋  (Fixes #359)
I’ve added a simple website — Linked Login Page — using HTML and CSS under the “Websites & Projects” section.
Kindly review and accept this PR as part of Hacktoberfest. Thank you!

**Comment by pranavpanmand** [comment]
### Hi @aayush105  👋
#359

I’ve also created another small project using only HTML and CSS — a Contact Us page. I wanted to ask if I can add it under the “Websites and UI” category as a new PR.
I noticed that in many projects like Portfolio or E-Shop, the Contact Us section is not currently enabled.

So, should I go ahead and add this Contact Us page in the Websites and UI category as a pull request?

**Comment by aayush105** [comment]
@pranavpanmand 

Firstly please fix the previour prs issue i have commented in your all prs. Check it once before proceeding to another prs.
